opengrok-web/src/main/webapp/xref.jspf

```diff
@@ -129,7 +129,7 @@ org.opengrok.indexer.web.QueryParameters"
                             defs, annotation, project);
                     } else if (g == AbstractAnalyzer.Genre.IMAGE) {
         %></pre>
-    <img src="<%= rawPath %>?<%= QueryParameters.REVISION_PARAM_EQ %><%= Util.URIEncode(rev) %>"/>
+    <img src="<%= rawPath %>?<%= QueryParameters.REVISION_PARAM_EQ %><%= Util.URIEncode(rev) %>" alt="<%= path %>"/>
```

Comment:
Maybe use `"Image from Source Repository"` to be consistent with https://github.com/oracle/opengrok/blob/539b5afffee12c5504fba86b2e8efbfe9f22839b/opengrok-web/src/main/webapp/xref.jspf#L180 ?

Reply:
That might do it, thanks.